project_tests.sh

```diff
@@ -1,9 +1,7 @@
 #!/bin/bash
-set -e
 
-virtualenv venv
-source venv/bin/activate
-pip install -r requirements.txt
+tox
+. .tox/py35/bin/activate
 pip install coveralls
-coverage run -m unittest discover tests
-COVERALLS_REPO_TOKEN=$(cat /etc/coveralls/tokens/elife-crossref-xml-generation) coveralls
+COVERALLS_REPO_TOKEN=$(cat /etc/coveralls/tokens/elife-article) coveralls
```

Comment:
This should maybe still use the ``elife-crossref-xml-generation`` coveralls token instead of the ``elife-article`` one?